integration-tests/src/main/java/eu/xenit/alfresco/healthprocessor/example/ExampleHealthProcessorPlugin.java

```diff
@@ -0,0 +1,78 @@
+package eu.xenit.alfresco.healthprocessor.example;
+
+import eu.xenit.alfresco.healthprocessor.plugins.api.SingleNodeHealthProcessorPlugin;
+import eu.xenit.alfresco.healthprocessor.reporter.api.NodeHealthReport;
+import eu.xenit.alfresco.healthprocessor.reporter.api.NodeHealthStatus;
+import java.util.Objects;
+import org.alfresco.model.ContentModel;
+import org.alfresco.service.cmr.repository.NodeRef;
+import org.alfresco.service.cmr.repository.NodeService;
+import org.alfresco.service.cmr.repository.StoreRef;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ExampleHealthProcessorPlugin extends SingleNodeHealthProcessorPlugin {
+
+    private static final Logger logger = LoggerFactory.getLogger(ExampleHealthProcessorPlugin.class);
+
+    private final NodeService nodeService;
+
+    /**
+     * For testing purposes, we keep track of the number of nodes this plugin has processed.
+     */
+    private long numberOfNodesProcessed;
+
+    public ExampleHealthProcessorPlugin(NodeService nodeService) {
+        this.nodeService = nodeService;
+    }
+
+    @Override
+    protected Logger getLogger() {
+        return logger;
+    }
+
+    @Override
+    protected NodeHealthReport process(NodeRef nodeRef) {
+        if (!StoreRef.STORE_REF_WORKSPACE_SPACESSTORE.equals(nodeRef.getStoreRef())) {
+            logger.debug("Ignoring '{}', since we are only interested in workspace://SpacesStore nodes", nodeRef);
+            return null;
+        }
+        numberOfNodesProcessed++;
+
+        if (!nodeService.exists(nodeRef) || nodeService.getNodeStatus(nodeRef).isDeleted()) {
+            return new NodeHealthReport(NodeHealthStatus.NONE, nodeRef, "Node does not exist or is deleted");
+        }
+        if (nodeService.hasAspect(nodeRef, ContentModel.ASPECT_TEMPORARY)) {
+            return new NodeHealthReport(NodeHealthStatus.UNHEALTHY, nodeRef);
+        }
+        NodeHealthStatus status = nodeService.hasAspect(nodeRef, ContentModel.ASPECT_TEMPORARY) ?
```

Comment:
will have returned in line 46 already if true

Reply:
You can just drop the if clause from 45 to 47

Reply:
Seems I forgot to cleanup after refactor :-)